htdocs/request_account/process_new_account.php

```diff
@@ -143,6 +144,28 @@
                      'Email'            => $from,
                      'CenterID'         => $site,
                     );
+        /*
+        foreach($_REQUEST as $k=>$v) {
+            error_log($k." -->  ".$v);
```

Comment:
@ridz1208 
user_accounts/test/user_accountsTest.php:184
        $this->_verifyUserModification(
            'user_accounts',
            'UnitTester',
            'First_name',
           // 'NewFirst' => 'Unit',
            'Unit'
        ); 
try it